blocks/blocks.js

```diff
@@ -208,6 +221,7 @@ const createValidators = () => {
   Blockly.Extensions.register('validate_STDDEV', _createNonNeg('STDDEV'))
 
   Blockly.Extensions.register('validate_DATE', validateDate('DATE'))
+  Blockly.Extensions.register('validate_COLOR', validateColor('COLOR'))
```

Comment:
`validateColor` requires a pattern but none is given.

Reply:
I'm using the same pattern now as the single column but also allow for an empty string in new code - see latest commits. Thanks for the read through!